docs/conf.py

```diff
@@ -142,6 +142,7 @@ def nothing(*arg):
 #     'whatsnew-1.8': 'index',
 #     'whatsnew-1.9': 'index',
 #     'whatsnew-1.10': 'index',
+#     'whatsnew-2.0': 'index',
```

Comment:
This change is OK. It appears in the `2.0-branch`, so it does not need to be backported.